src/responses/live.post-live-thumbnails.response.ts

```diff
@@ -0,0 +1,4 @@
+export interface LivePostLiveThumbnailsResponseRootObject {
+  thumbnails: any[];
```

Comment:
Are there no types for this property? I'd guess this is an array of image-versions.

Reply:
for me it returns an array of urls, you can try with anyone's livestream, it seems to work not only for the owner and works during the live

Reply:
> 
> 
> for me it returns an array of urls, you can try with anyone's livestream, it seems to work not only for the owner and works during the live

But then it's a `string[]` instead of an `any[]`

Reply:
Ok, I will change it.